test/Microsoft.ML.CpuMath.UnitTests.netcoreapp/UnitTests.cs

```diff
@@ -77,431 +81,661 @@ public CpuMathUtilsUnitTests()
             _testDstVectors = new AlignedArray[] { testDstVectorAligned1, testDstVectorAligned2 };
         }
 
+        private static void CheckProperFlag(string mode)
+        {
+#if NETCOREAPP30
```

Comment:
So if we are running tests on non-.NET Core 3.0 (either full fx or 2.1), does this mean we are running the same test 3 times in a row? Is it worth only running it one time in those environments?